cltk/tests/test_corpus.py

```diff
@@ -786,5 +786,12 @@ def test_swadesh_old_norse(self):
         match = swadesh.words()[0]
         self.assertEqual(first_word, match)
 
+    def test_swadesh_arabic(self):
+        swadesh = Swadesh('ar')
+        first_word = ' أنا‎'
```

Comment:
Don't forget The same here use `""` instead of `''` and remove additional space in the string :)))